### PR TITLE
Added support for endpoint /merchantsLocations/nearMe

### PR DESCRIPTION
**Pull request by rui-castro**
## Description

Added support for endpoint /merchantsLocations/nearMe https://obx10-api.fftech.info/swagger/ui/index#!/MerchantLocations/MerchantLocations_FindNearMe

@joamag @tsilva

### CODE REVIEW

**Comment by joamag** [comment]
@tsilva take a look at this

**Comment by joamag** [comment]
@tsilva bump

File: src/farfetch/merchant_location.py [hunk 1]
```diff
@@ -40,9 +40,20 @@
 class MerchantLocationApi(object):
 
     def list_merchant_locations(self, merchant_id = None):
-        url = self.base_url + "merchantslocations"
+        if merchant_id:
+            url = self.base_url + "merchantsLocations/%s" % merchant_id
```
Comment (tsilva) [inline src/farfetch/merchant_location.py]: self.base_url + "merchantsLocations/" + str(merchant_id) (for consistency)

Comment (tsilva) [inline src/farfetch/merchant_location.py]: put in same line after if merchant_id:

File: src/farfetch/merchant_location.py [hunk 2]
```diff
@@ -40,9 +40,20 @@
 class MerchantLocationApi(object):
 
     def list_merchant_locations(self, merchant_id = None):
-        url = self.base_url + "merchantslocations"
+        if merchant_id:
+            url = self.base_url + "merchantsLocations/%s" % merchant_id
+        else:
+            url = self.base_url + "merchantsLocations"
```
Comment (tsilva) [inline src/farfetch/merchant_location.py]: put in same line after else:

**Pull request merged**